frontend/app/views/fragments/pricing/billingPeriodChoice.scala.html

```diff
@@ -47,6 +47,9 @@
             generateChargeAmount: function() {
                 return formattedDiscountedCharge(checkoutForm.currency, this.amount, this.discount);
             },
+            generateDisplayAmount: function() {
```

Comment:
There's starting to be a bit of repetition here, duplicated between monthly & annual billing. Perhaps some of it is removable?! (in a subsequent PR, not necessarily for this one)